Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -3352,6 +3357,7 @@ static ASCIILiteral compositingReasonToString(CompositingReason reason)
     case CompositingReason::Root: return "root"_s;
     case CompositingReason::Model: return "model"_s;
     case CompositingReason::BackdropRoot: return "backdrop root"_s;
+    case CompositingReason::AnchorPositioning: return "anchor positioning"_s;
```

Comment:
I think these reasons propagate out into web inspector code (they show in the Layers tab) so I suspect there is more to do here.